src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDirectiveType.cs

```diff
@@ -16,6 +16,11 @@ public enum TagHelperDirectiveType
         /// <summary>
         /// A @removeTagHelper directive.
         /// </summary>
-        RemoveTagHelper
+        RemoveTagHelper,
+
+        /// <summary>
+        /// A @tagHelperPrefix directive.
```

Comment:
nit (because comment is currently consistent with that of other values): `<c>@tagHelperPrefix</c>`